ops_utils/tdr_utils/tdr_api_utils.py

```diff
@@ -180,6 +182,58 @@ def delete_files(
             check_interval=check_interval
         ).run()
 
+    def _delete_snapshots(self, dataset_id: str, file_ids: set[str]) -> None:
```

Comment:
Could this be renamed to be specific to deleting snapshots that contain specific files? It is a little confusing when class also has the method `delete_snapshots`